tests/baselines/reference/ambientErrors.types

```diff
@@ -10,13 +10,13 @@ declare var x = 4;
 
 // Ambient functions with invalid overloads
 declare function fn(x: number): string;
->fn : { (x: number): string; (x: "foo"): number; }
+>fn : { (x: number): string; (x: 'foo'): number; }

```

Comment:
Quite interesting that we already marked these nodes as reused.

Reply:
Even reused nodes are cloned, and the the delimiters are replaced during cloning. The markers  reflect if the node has an original source node it can be traced back not so much node instance reuse.